pkg/auth/handler/ssohandler/config.go

```diff
@@ -0,0 +1,57 @@
+package ssohandler
+
+import (
+	"net/http"
+
+	"github.com/skygeario/skygear-server/pkg/auth"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authz"
+	"github.com/skygeario/skygear-server/pkg/core/auth/authz/policy"
+	"github.com/skygeario/skygear-server/pkg/core/config"
+	"github.com/skygeario/skygear-server/pkg/core/handler"
+	"github.com/skygeario/skygear-server/pkg/core/server"
+)
+
+func AttachConfigHandler(
+	server *server.Server,
+	authDependency auth.DependencyMap,
+) *server.Server {
+	server.
+		Handle("/sso/config", &ConfigHandler{}).
+		Methods("OPTIONS", "POST")
+	return server
+}
+
+type ConfigHandler struct {
+}
+
+// NewHandler returns the SSO configs.
+//
+// curl \
+//   -X POST \
+//   -H "Content-Type: application/json" \
+//   -H "X-Skygear-Api-Key: API_KEY" \
+//   -d @- \
+//   http://localhost:3000/sso/config \
+// <<EOF
+// {
+// }
+// EOF
+//
+// {
+//     "result": [
+//         "http://localhost",
+//         "http://127.0.0.1"
+//     ]
```

Comment:
IMO

1. If we want this api extensible in future, we can use `/sso/config` a more generic name. But we may want to put the result in a dictionary. So that we can add new key with other new information, and keep the api compatible with the old version SDK, this is important. The payload may looks like this.
    ```
    {
        "result": {
            "authorized_urls": [
                "http://localhost",
                "http://127.0.0.1"
            ]
        }
    }
    ```

1.  If we plan to keep this api simple and will not extent it in the future. For any new feature that the SDK need to get config from server, we will add new api. We should rename this api to a more specific name like what you mentioned `/sso/authorized_urls`. And the payload can be like this.

    ```
    {
        "result": [
            "http://localhost",
            "http://127.0.0.1"
        ]
    }
    ```

What do you think?

Reply:
@carmenlau I updated response structure, I consider `/sso/config` should have a more generic response. Please help review. Thanks.